.babelrc

```diff
@@ -1,8 +1,3 @@
 {
-  "plugins": [
-    "transform-class-properties",
-    "transform-object-assign",
-    "transform-object-rest-spread"
-  ],
-  "presets": ["es2015", "react"]
+  "presets": ["env", "react"]
```

Comment:
If I remember correctly they deprecated a lot of the old presets in favor of the `env` preset. I like to see it here :)